### ISSUE TITLE
FAQ: What is Carvel?

### ISSUE THREAD

**Issue by jonasmit** [kube-academy/backlog#9]

**Comment by djschny** [comment]
Perhaps start by seeing if we can use the content/idea from this point in a CNCF talk the Carvel team gave:

https://youtu.be/Kd1sWySJGhc?t=179

**Comment by eitansuez** [comment]
i agree, helen does an excellent job of summarizing carvel succinctly in that video.  thanks.

**Comment by djschny** [comment]
I reviewed the carvel video as requested and unfortunately yes it is quite long (8 minutes) which is as long or longer than many lessons in other courses.

My suggestion would be to conclude after the `Philosophy` slide. This would put the video at just over 2 minutes.

**Comment by jonasmit** [comment]
It’s hard but imagine you are in an elevator and your goal is to get
someone excited about googling more after you answer the question and the
doors open.  Not to provide all the info.

**Comment by eitansuez** [comment]
i got it down to six minutes.  take-3 has been uploaded to google drive.

**Comment by lxie123** [comment]
@meghnam00 Has this been published?
https://drive.google.com/drive/folders/1KtFeDJnJUpMLTobilXgsLSME_QHWZCKC
